include/defines.h

```diff
@@ -21,23 +22,44 @@
   SOFTWARE.
 */
 
-#pragma once
+/**
+ * @file defines.h
+ * @brief Defines default values used in the libeconf library.
+ */
 
-/* --- defines.h --- */
+#pragma once
 
 #include "../include/helpers.h"
 
-/* Defines default values used in the library  */
-
-/* Default econf_file length on creation */
+/**
+ * @brief Default econf_file length on creation.
+ */
 #define KEY_FILE_DEFAULT_LENGTH 8
 
-/* NULL value */
+/**
+ * @brief Default NULL value.
+ */
 #define KEY_FILE_NULL_VALUE "_none_"
+
+/**
+ * @brief Default NULL value hash.
+ */
 #define KEY_FILE_NULL_VALUE_HASH hashstring(KEY_FILE_NULL_VALUE)
```

Comment:
These two could also need a better description - as far as I remember these are even user visible in the configuration files. If that's still the case it would be worth mentioning that.